brushfire-core/src/main/scala/com/stripe/brushfire/Tree.scala

```diff
@@ -1,14 +1,14 @@
 package com.stripe.brushfire
 
+import com.stripe.bonsai.FullBinaryTreeOps
 import com.twitter.algebird._
 
-// type Tree[K, V, T] = AnnotatedTree[K, V, T, Unit]
```

Comment:
Any reason to get rid of this type alias? I think we should either keep it, or rename `AnnotatedTree` to `Tree`, as in @NathanHowell's PR.